server/controllers/boards.js

```diff
@@ -0,0 +1,158 @@
+const asyncHandler = require("express-async-handler");
+const Board = require('../models/Board');
+const Card = require('../models/Card');
+const Column = require('../models/Column');
+
+exports.createBoard = asyncHandler(async (req, res) => {
+  const { title } = req.body;
+  if (title) {
+    try {
+      const inProgress = new Column({ title: 'In Progress' });
+      const completed = new Column({ title : 'Completed' });
+      
+      await inProgress.save();
+      await completed.save();
+
+      const board = new Board({ title, columns: [inProgress._id, completed._id]});
+      const doc = await board.save();
+
+      res.status(201).json({
+        success: {
+          boardId: doc._id,
+        }
+      });
+    } catch (err) {
+      res.status(500);
+      console.error(err);
+    }
```

Comment:
With this try/catch here would it still go forward with express-async-handler and return a JSON response of the error?

Logging is good, but want to mention we don't necessarily need try/catch unless we are doing more fine tune error handling otherwise all errors are caught by express-async-handler.

Reply:
Okay! I'll remove them since they're not needed then.